src/components/common/fetchData.js

```diff
@@ -30,41 +31,70 @@ export async function getAllContentList() {
   return data.list;
 }
 
+export async function getCategoryList() {
+  const res = await fetch(
+    `${process.env.NEXT_PUBLIC_BASE_URL}/rcms-api/1/category/list`,
+    { cache: "no-store" }
+  );
+  const data = await res.json();
+  return data.list;
+}
+
 export async function getDetails(id) {
   const res = await fetch(
-    `${process.env.NEXT_PUBLIC_BASE_URL}/rcms-api/1/content/details/${id}`,{ cache: "no-store" }
+    `${process.env.NEXT_PUBLIC_BASE_URL}/rcms-api/1/content/details/${id}`,
+    { 
+      method: "GET",
+      headers: { "Content-Type": "application/json" },
+      credentials: "include",
+      cache: "no-store", 
+      revalidate: 0,
```

Comment:
please tell me (maybe I'm ignorant for this option),
'no-store' with revalidate option is valid?

Reply:
I had to revalidate it once to force clear cache fetching. 
Not necessary now. And yes revalidate with no-store is redundant. 👍